README.md

```diff
@@ -1 +1,46 @@
-# Newsweek_bootstrap
\ No newline at end of file
+# bootstrap-project
+
+> This is a pure bootstraps project and it is responsive but it also involues little css and media queries.  
+
+![screenshot](./bootstrap.png) 
```

Comment:
- [ ]  Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...

Reply:
I have fix this issue. i made the readme more descriptive